roxctl/generate/netpol/netpol.go

```diff
@@ -42,6 +44,8 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 			return generateNetpolCmd.generateNetpol()
 		},
 	}
+	c.Flags().BoolVar(&generateNetpolCmd.treatWarningsAsErrors, "strict", false, "treat warnings as errors")
```

Comment:
Cannot add commnet on line 62 but why not using https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive

Reply:
We used to have it like this but the auto-generated error message was unreadable. I will find it and show an example in a moment.

Reply:
Here: https://github.com/stackrox/stackrox/pull/2602#pullrequestreview-1063436976

Reply:
Using the `MarkFlagsMutuallyExclusive ` would give us:

```
ERROR:	if any flags in the group [output-dir output-file] are set none of the others can be; [output-dir output-file] were all set
```

which I find really difficult to read.